bin/pycbc_live

```diff
@@ -574,8 +574,12 @@ parser.add_argument('--output-background-n-loudest', type=int, default=10000,
 parser.add_argument('--newsnr-threshold', type=float, default=0)
 parser.add_argument('--max-batch-size', type=int, default=2**27)
 parser.add_argument('--store-loudest-index', type=int, default=0)
-parser.add_argument('--max-psd-abort-distance', type=float, default=numpy.inf)
-parser.add_argument('--min-psd-abort-distance', type=float, default=-numpy.inf)
+parser.add_argument('--max-psd-abort-distance', type=float, default=numpy.inf,
+                    help="Safety BNS horizon distance (in Mpc) above which a "
+                    "detector's data is discarded.".)
```

Comment:
Thanks for the contribution @rmacas! There is a typo here (extra period after the end of the string) which is making the tests fail.

Reply:
Thanks @titodalcanton, well spotted. Fixed the typo, now the PR passes the checks.